Documentation/supported-mod-list.md

```diff
@@ -883,365 +903,295 @@ On **android** install launcher from [here](https://github.com/FWGS/XashXT/relea
 5. [They Hunger: Lab of Horrors](https://yadi.sk/d/V_us_m78pFaMK) by *Jordan*
 6. [They Hunger: Route666 Demo](https://yadi.sk/d/XKWCseAPohkG6) by *MadMapper*
 
-# List of games and mods with custom gamedll
-For mods from this category - first of all place mod folder to folder where is **valve** folder located.
-
-On **android** need to install separate launcher with custom game libraries
-
-On **iOS** need to run the game from Xash3D FWGS app with command-line parameter `-game <modfolder>` and special library suffix(if it needed).
-
-On **Linux** and **OS X** need to download special build for this OSes.
-
-Package for **FreeBSD** already has libraries for some mods from this list.
-
-NOTE: Many mods from this list was ported only on android, but you can use source code to build mod libraries manually for your platform.
+# List of They Hunger-based mods
+To run this mods on specific platforms you may be need to compile libraries from **theyhunger** branch of [hlsdk-portable](https://github.com/FWGS/hlsdk-portable).
 
-1. [Adrenaline Gamer](https://www.moddb.com/mods/adrenaline-gamer)
+1. [They Hunger: Rebuild](https://www.moddb.com/mods/they-hungerrebuild) - This mod uses new **func_vehicle** entity and our **hlsdk-portable** libraries.
 
-Android launcher: [ModDB](https://www.moddb.com/mods/adrenaline-gamer/downloads/adrenaline-gamer-66-android-port), [Github](https://github.com/nekonomicon/AdrenalineGamer-android/releases)
-
-On **Linux** you can use *OpenAG* client by *YaLTeR*
-
-Source code:
-
-Original source code: https://github.com/martinwebrant/agmod
+# List of supported mods from mobile_hacks branch of HLSDK Portable by FWGS
```

Comment:
I think it might be good to note that we ship this branch by default on `android`, `nswitch`, `psvita`.